pom.xml

```diff
@@ -25,12 +25,12 @@
 		<module>doc-snippets</module>
 		<module>tutorial</module>
 		<module>astrix-all</module>
+		<module>astrix-bom</module>
 		<module>astrix-modules</module>
 		<module>astrix-metrics</module>
 		<module>astrix-contracts</module>
 		<module>astrix-netty-remoting</module>
 		<module>astrix-test-support</module>
-		<module>hystrix-multiconfig</module>
```

Comment:
The contents of `hystrix-multiconfig` has been removed previously, and replaced with a with `<relocation>`. If  `hystrix-multiconfig` is removed as a module in this branch, then we should also consider removing `hystrix-multiconfig/pom.xml`.

Reply:
Great point, removing it!